src/components/ShareCurrentScenarioButton/ShareCurrentScenarioButton.js

```diff
@@ -22,6 +24,11 @@ const ShareCurrentScenarioButton = () => {
   } = useShareCurrentScenarioButton();
 
   const { isDirty } = useFormState();
+  if (isDirty && shareScenarioDialogLabels?.button?.tooltip)
```

Comment:
i think it's better to modify getShareScenarioDialogLabels for get the good label

Reply:
Maybe, it's better to change it in ui package and add a disabled tooltip label there ? :slightly_smiling_face:  (or make like in CreateScenarioLabels in the function itself, yes)

Reply:
Done, thank you for the feedback :slightly_smiling_face: 
I moved the call to the `useFormState` hook in the `ShareCurrentScenarioButtonHook` to get the value of `isDirty`, and added it as a parameter of the `getShareScenarioDialogLabels` function.

Modifying the UI component would mean to add a new prop for the form `isDirty` state to choose between possible tooltips labels: it's probably best to keep this logic in the webapp repo and provide only the tooltip message to the generic component.